TAC/Meetings/2022/2022-04-07/CCC_TAC_Minutes-2022-04-07.md

```diff
@@ -10,15 +11,15 @@
 
 * **Zoom**: [https://zoom.us/j/184384055](https://zoom.us/j/184384055) (pw: 43690)
 * **Calendar**: [https://calendar.confidentialcomputing.io](https://calendar.confidentialcomputing.io) ([ical](https://calendar.google.com/calendar/ical/c_c0pcihr7n2n1k3a38i32d9ag10%40group.calendar.google.com/public/basic.ics), [Google Calendar](https://calendar.google.com/calendar/u/0/r?cid=c_c0pcihr7n2n1k3a38i32d9ag10@group.calendar.google.com))
-* **Recording**: [YouTube TAC meetings](https://www.youtube.com/playlist?list=PLmfkUJc39uMjaB_I1dYW72I44kr9QzG_B)
+* **Recordings**: [YouTube Playlist](https://www.youtube.com/playlist?list=PLmfkUJc39uMjaB_I1dYW72I44kr9QzG_B)
 
 ### Voting members of the TAC
 
 *Per the [charter](https://charter.confidentialcomputing.io), all [CCC Premier members](https://confidentialcomputing.io/members/) receive one vote on the TAC. Quorum for votes is at least 50% of voting members present.*
 
 * **Accenture** - Giuseppe Giordano
-* **Ant Group** - Zongmin Gu / Tate, Hongliang Tian
-* **Arm** - Thomas Fossati / Michael Lu
+* **ANT Group** - Zongmin Gu
+* **ARM** - Thomas Fossati / Michael Lu
```

Comment:
Revert.  The correct capitalization is "Arm".  See https://www.arm.com/company/policies/trademarks

And according to https://en.wikipedia.org/wiki/Ant_Group "Ant Group" was also correct as it was.